combine_projections.py

```diff
@@ -0,0 +1,30 @@
+from os import listdir, path
+import csv
+
+CUR_PROJ = 'current-projections.csv'
+
+def combine_projections():
+    all_projections = filter(lambda x: x.split('-')[0] == 'current' and x != CUR_PROJ , listdir('data'))
+    if len(all_projections) > 0:
+        hold = {}
+        for proj_file in all_projections:
+            with open(path.join('data', proj_file), 'rb') as csvfile:
+                csvdata = csv.DictReader(csvfile)
+                for idx, row in enumerate(csvdata):
+                    if idx > 0:
+                        if row['playername'] in hold:
+                            hold[row['playername']].append(row['points'])
+                        else:
+                            hold[row['playername']] = [row['points']]
+    proj_dict = {k: v for k, v in hold.iteritems() if len(hold[k]) == len(all_projections)}
```

Comment:
Also only fear for me here is that we need a good way to normalize names if any sources give us slightly different names in the future, because it filters any names that don't appear in all the projection files. So if capitalization is off or something random, we could lose players